shared/aux.go

```diff
@@ -85,8 +86,8 @@ func BasePath() string {
 	return filepath.Join(filepath.Dir(b), "../..")
 }
 
-// EnvDir returns the environment directory of the project based on the package passed.
-func EnvDir(pkg string) (string, error) {
+// envDir returns the environment directory of the project based on the dir package passed.
+func envDir(dir string) (string, error) {
```

Comment:
but then in each pkg you call , you will need to add different path

this is just a nicer user friendly way to do it 
 
and the base path it was already being used in a lot of places reuturning to actual home "../.."

this one is on the root project

Reply:
https://github.com/rancher/distros-test-framework/blob/main/shared/aux.go#L81-L82
> // BasePath returns the base path of the project.

I think that is the root of the project also right?

Reply:
no its not, i will update this doc comment

Reply:
It should be. https://pkg.go.dev/runtime#Caller
> The argument skip is the number of stack frames to ascend, with 0 identifying the caller of Caller.

Meaning that this returns the string for the file aux.go, then we just need to go up one level to get to the base path. This function today actually has a bug in that it goes up two levels, so it is returning the path directly before `distros-test-framework` but every function that calls it is then appending `/distros-test-framework`.

I think we should actually fix this bug, fix the functions that are using it, and just use this to always get the correct path instead of the extra `case` logic in this `envDir`

Reply:
got it , fixed